time stamp
float32 leftAngle
float32 leftVel
float32 rightAngle
float32 rightVel
